### ISSUE TITLE
Cannot reproduce results

### ISSUE THREAD

**Issue by Nicolas99-9** [jcwleo/mario_rl#24]
I tried to run mario_a2c.py, mario_ppo.py and mario_curio.py but for non of them I cannot improve the reward.
Did you use the same hyper-parameters as in the files to conduct the evaluation? (i.e. number of workers, learning rate)
Which version of the libraries did you use ?

For instance, A2C without ICM: (after 3M time-steps)

![Screenshot from 2019-08-06 16-38-51](https://user-images.githubusercontent.com/16637853/62520917-83927080-b869-11e9-8d79-24a620e366a7.png)
